lms/urls.py

```diff
@@ -173,7 +173,7 @@
         url(r'^api/badges/v1/', include('badges.api.urls', app_name='badges', namespace='badges_api')),
     ]
 
-js_info_dict = {
+js_info_dict  =  {
```

Comment:
I put a dummy change in here just to draw attention to it, because my editor complained about this code being unused.  Was wondering if it's obsolete cruft that should be deleted, but wanted to get an extra set of eyes on it before I "clean up" something I shouldn't.

Reply:
Not sure, but there don't seem to be any references to `js_info_dict` in this repo. Based on [these docs](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#the-javascript-catalog-view), it seems like it's probably no longer used, if it's not referenced in any URL pattern.